modules/migration-manually-rolling-back-migration.adoc

```diff
@@ -4,12 +4,75 @@
 // * migration/migrating_4_2_4/troubleshooting-4-2-4.adoc
 
 [id='migration-manually-rolling-back-migration_{context}']
-= Manually rolling back a migration
+= Rolling back a migration
 
 If your application was stopped during a failed migration with the {mtc-full} ({mtc-short}), you must roll it back manually in order to prevent data corruption in the persistent volume.
 
-This procedure is not required if the application was not stopped during migration, because the original application is still running on the source cluster.
+Rollback is not required if the application was not stopped during migration because the original application is still running on the source cluster.
 
+ifeval::["{mtc-version}" >= "1.4"]
```

Comment:
I'm not as familiar with conditionalizing based on version numbers and how that renders within the docs. I'll lean on Kathryn for a little help here.

Reply:
It works and is how we normally do this.